userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -132,7 +142,8 @@ User:: A person who logs on to Kaui.
 
 == X
 
+XML::
+
 == Y
 
 == Z
-
```

Comment:
Missing `-` (Not sure if it is important, but just making a note)

Reply:
It's okay